docroot/themes/humsci/humsci_basic/src/js/main-nav/menu-toggle.js

```diff
@@ -0,0 +1,56 @@
+const menuToggle = document.querySelector('.su-main-nav__toggle');
+const mainMenu = document.querySelector('.su-main-nav__menu-lv1');
+const mobileNavBreakpoint = 992;
+let windowWidth = window.innerWidth;
+let isHidden;
+let wasDesktopSize;
+
+const collapseNav = () => {
+  menuToggle.setAttribute('aria-expanded', false);
+  mainMenu.setAttribute('aria-hidden', true);
+  menuToggle.innerHTML = "Menu";
+  isHidden = true;
+}
+
+const showNav = () => {
```

Comment:
I tried out a DRY-er function for these show/CollapseNav functions, as well as the ones in nested-toggler:

if you want to make a new file called changeNav.js in this same main-nav directory, add the following:


```
const changeNav = (toggle, parent, isOpen) => {
  toggle.setAttribute('aria-expanded', isOpen);
  parent.setAttribute('aria-hidden', !isOpen);
// just set isHidden variable in the local file underneath the call to this function
}

export default changeNav;

```

and then to import it and use it in **this file** you can do:

```
import changeNav from './changeNav';


// rather than calling showNav, you can just do:

 // Toggle the nav when the the button is clicked
  menuToggle.addEventListener('click', () => {
    if (isHidden) {
      changeNav(menuToggle, mainMenu, true);
      isHidden = false;
    } else {
      changeNav(menuToggle, mainMenu, false);
      isHidden = true;
    }
  });
```